crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -0,0 +1,15 @@
+use restate_core::network::NetworkError;
```

Comment:
1) License header missing
2) in mod files, please define all modules before imports.